content/blog/2020/05/2020-05-31-machine-learning-plugin-community-bonding.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Machine Learning Plugin project - community bonding report"
+tags:
+- machinelearning
+- datascience
+- plugins
+- communitybonding
+- gsoc
+- gsoc2020
+author: loghijiaha
```

Comment:
Maybe you also want to add an opengraph images and description. It will make the blogpost more visible. https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post

Reply:
Thank you for the suggestions. I will add those changes.